src/p2p/ServiceQueue.ts

```diff
@@ -432,6 +507,28 @@ export function sendRequests(): void {
   }
   addProposals.length = 0
   removeProposals.length = 0
+
+  for (const dropNGTInfo of debugDropNGTs) {
+    const unsignedRemoveNetworkTx = {
+      txHash: dropNGTInfo.txHash,
+      cycle: dropNGTInfo.usePrevCycle ? CycleChain.newest.counter - 1 : CycleChain.newest.counter,
+    }
+    txRemove.push(unsignedRemoveNetworkTx)
+
+    Comms.sendGossip(
```

Comment:
is it possible that `Comms.sendGossip` throws error? if so, shouldn't it be caught & logged? with this implementation, what happens in error case? Do we ignore all the NGTs to be dropped? 

same question goes [here](https://github.com/shardeum/shardus-core/pull/353/files#diff-0fd11e27abb2fd84b758cf52d189a7661521d10f3dd2c035771a04223fceecadR226) also

Reply:
Comms.sendGossip has a try catch inside it that already handles errors